backend/spec/features/admin/orders/order_details_spec.rb

```diff
@@ -111,11 +108,14 @@
         expect(page).to have_content("Default")
       end
 
-      it "will show the variant sku" do
-        order = create(:completed_order_with_totals)
-        visit spree.edit_admin_order_path(order)
-        sku = order.line_items.first.variant.sku
-        expect(page).to have_content("SKU: #{sku}")
+      context "with a completed order" do
+        let!(:order) { create(:completed_order_with_totals) }
```

Comment:
Using let is better for composability and is always preferred. This let actually overrides a `let!(:order)` higher up in the file, which the local definition would not do. This is a good thing.